### ISSUE TITLE
Sort directory of channels by latest message

### ISSUE THREAD

**Issue by BTLIX** [RocketChat/feature-requests#120]
In the current directory of channels you are able to sort based on number of users inside a channel and based on when a channel was created. I think it is imperative to be able to sort based on last message (last activity). That's what people may often be looking for - where are the discussion and communication currently ongoing.

**Comment by Polgabarro** [comment]
Agree. It's posible to receive a property with the date of last message? It's useless to know what is the date that the channel was created.
